demos/Shrine/Shrine/ShrineCollectionViewController.swift

```diff
@@ -64,9 +64,15 @@ class ShrineCollectionViewController: UICollectionViewController {
   func collectionView(_ collectionView: UICollectionView,
                       layout collectionViewLayout: UICollectionViewLayout,
                       sizeForItemAtIndexPath indexPath: IndexPath) -> CGSize {
-      let cellWidth = floor((self.view.frame.size.width - (2 * 5)) / 2) - (2 * 5)
-      let cellHeight = cellWidth * 1.2
-      return CGSize(width: cellWidth, height: cellHeight)
+    var safeAreaInset: CGFloat = 0
+    #if swift(>=3.2)
+      if #available(iOS 11.0, *) {
+        safeAreaInset = self.view.safeAreaInsets.left
+      }
+    #endif
+    let cellWidth = (floor((self.view.frame.size.width - (2 * 5)) / 2) - (2 * 5)) - safeAreaInset
```

Comment:
Consider performing the multiplication here (2 * 5 = 10) or else extract into constants.